client/web/src/models/ILabel.ts

```diff
@@ -0,0 +1,10 @@
+export interface ILabel {
+  "id": string,
```

Comment:
you remove double-quote and virgule in entire file

Reply:
Done